src/about-us/how-to-raise-issues-and-share-feedback.md

```diff
@@ -0,0 +1,118 @@
+
+### dxw monthly connect meetings
+
+Monthly connect meetings open to all staff are an opportunity to discuss any issues relating to the company with a couple of the Directors. These are less formal multi-way conversations which are sometimes themed in response to current issues or might have a completely open agenda.
+
+## Where do I go if I want to raise something
+
+In addition to the above channels, there are a number of ways you can raise issues, make suggestions, and ask questions of the company.
+
+### Strategic/owner issues
+
+Strategic, or owner-related, issues are usually dealt with to a longer time frame and include things like:
+
+* things that impact business performance
+* ideas that might improve business performance (for example, in terms of growing income or reducing costs)
+* issues that might pose a meaningful risk to the success of the business (including anything impacting colleague morale)
+* questions on business strategy, risk management and strategic decision making
+
+The Trust Board and Staff Council have a specific role in understanding and sharing staff insight, ideas and concerns relating to dxw's business strategy and other ownership-related issues.
+
+An important role of the Trustees is to make sure that everyone has the opportunity to share their views on strategic topics as owners. Trustees listen, reflect and can use this input to inform how they carry out their oversight of the Company Board. The Staff Council supports them in this role by gathering and sharing staff feedback on strategic issues, along with their own thoughts and insights.
+
+The Trustees have quarterly formal meetings with representatives of the Company Board where they raise important matters as part of their oversight function and make sure strategic decisions are made with employee owners' views in mind. Where issues are important and urgent they will raise them as needed in between these meetings.
+
+### Operational issues
+
+Operational issues are things relating to the day-to-day running of the company, where you can expect to raise things as needed and receive timely feedback. We're doing it right if this feels like an open and respectful ongoing conversation about how the business is operating.
+
+The company will provide regular feedback on the main themes being raised by staff, any action being taken as a result and if not, why that's the case. The main channels for this feedback to staff will be dxw quarterly performance updates and the People Experience roadmap meetings. Additional updates will be provided via the Company weeknote and Bikesheds (internal blog posts), as and when needed.
+
+If you feel that you're not getting feedback on an issue you've raised, please let the Head of People know. They are responsible for making sure issues raised via different routes are recorded, have an allocated owner and are responded to in a timely and appropriate way.
+
+#### Who to approach about what
```

Comment:
question/suggestion: is this supposed to be only relevant to Operational issues, or could it be an h3?

Reply:
Per Slack, I've confirmed the current nesting is correct with Wendy